app/config/config.yml

```diff
@@ -36,7 +36,7 @@ framework:
     csrf_protection: ~
     validation:      { enable_annotations: true }
     templating:
-        engines: ['twig']
+        engines: ['twig','php']
```

Comment:
This change makes me cry :sob: Is there any chance to avoid it?

I'm starting to think that this paginator is seriously flawed or deprecated (because of this change, the other one in Composer, the impossibility of using the paginator in the entity to avoid polluting the controller, etc.)